challenge.md

```diff
@@ -5,19 +5,24 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone "SSH clone URL"
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+git diff
```

Comment:
We were really looking for `git status` here, but given how the question is worded, `git diff` is a reasonable answer - I won't take any points off. When you're actually working, you'll probably be using both `git status` *and* `git diff` to inventory your changes before making a commit.